src/systems/tips.js

```diff
@@ -145,6 +158,18 @@ const VALIDATORS = {
     if (userinput.activeSets.has(sets.cursorHoldingInteractable)) return FINISH;
     return VALID;
   },
+  object_rotate: function(userinput, scene, mediaCounter, store) {
+    if (!scene.is("frozen")) return INVALID;
+    if (mediaCounter.count() === 0) return INVALID;
+    if (store && store.state.activity.hasRotated) return FINISH;
```

Comment:
Probably makes sense to do this check first otherwise you are keeping this tip unfinished longer than it needs to be